packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -119,6 +120,7 @@ const MIN_GAS_PRICE_IN_GWEI = parseInt(env.MIN_GAS_PRICE_IN_GWEI, 10) || 0
 const MAX_GAS_PRICE_IN_GWEI = parseInt(env.MAX_GAS_PRICE_IN_GWEI, 10) || 70
 const GAS_RETRY_INCREMENT = parseInt(env.GAS_RETRY_INCREMENT, 10) || 5
 const GAS_THRESHOLD_IN_GWEI = parseInt(env.GAS_THRESHOLD_IN_GWEI, 10) || 100
+const RUN_METRICS_SERVER = env.RUN_METRICS_SERVER || 'true'
```

Comment:
True by default? Maybe it should depend on the `NODE_ENV`?

Reply:
I don't think coupling it to `NODE_ENV` makes sense, it should be explicit